livekit_agent.proto

```diff
@@ -74,25 +78,49 @@ enum JobStatus {
     JS_FAILED = 2;
 }
 
+message SimulateJobRequest {
+    JobType type = 1;
+    string room_id = 2;
+    string participant_id = 3;
+}
+
+message PingRequest {
```

Comment:
It's rather unexpected for Ping to signal more than liveliness. If we are updating the server about information.. I think we should rename it to something like `UpdateStatus`.

How do we tell the server that the worker is draining? (i.e. still alive and processing jobs, but don't send it additional ones)

What about updating the server about the current jobs it's processing?

Reply:
Added WorkerStatusUpdate.
Also do we want to more WorkerState? 
Just thinking out loud maybe the worker could just decline the job inside the request_handler